Velaptor/Reactables/Core/Reactable.cs

```diff
@@ -42,6 +43,28 @@ public virtual IDisposable Subscribe(IReactor<TData> reactor)
         /// <inheritdoc/>
         public abstract void PushNotification(TData data, bool unsubscribeAfterProcessing = false);
 
+        /// <inheritdoc/>
+        public void EndNotifications()
+        {
+            if (this.notificationsEnded)
+            {
+                return;
+            }
+
+            // ReSharper disable ForCanBeConvertedToForeach
+            /* Keep this for loop as a for-loop.  Do not convert to for-each.
```

Comment:
Do you need to specify "for" to describe the loop?  
```suggestion
            /* Keep this loop as a for-loop.  Do not convert to for-each.
```